src/main/java/uk/gov/ons/census/fwmt/jobservice/dto/rm/FieldworkFollowup.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.ons.census.fwmt.jobservice.dto.rm;
```

Comment:
Are we no longer going to keep these in separate lib or is it awaiting move while you work on repo: